website/blog/2023-01-25-ferretdb-fetches-data-query-pushdown.md

```diff
@@ -98,7 +98,7 @@ This is needed for our internal representation of the documents called `pjson`,
 With these documents, FerretDB can parse the data to internal `pjson` type, and iterate through all of them to apply the filters.
 After this process, as only one document matches the filter, the projection will be applied to it, so only the `active` (and `_id`) fields will be returned:
 
-```json5
+```json
```

Comment:
We used `json5` because that's not a valid JSON: unquoted field names can't rendered correctly by `json`. We should enable `json5`, not change it to `json`.